scripts/extract-registration-email.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+EMAIL_HTML=$(curl --silent 'localhost.localstack.cloud:4566/_aws/ses?email=support@cubecobra.com' | jq '.messages[0].RawData' | sed -e 's/=\\r\\n//g')
```

Comment:
Helper to extract the registration email
<img width="1185" height="74" alt="image" src="https://github.com/user-attachments/assets/527b58c3-b469-4711-ab42-94c0ee2e4bf6" />